<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="mainWidget">
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>110</y>
      <width>301</width>
      <height>351</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="4" column="0">
      <widget class="QPushButton" name="onBtn">
       <property name="text">
        <string>ON</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QPushButton" name="offBtn">
       <property name="text">
        <string>OFF</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0" colspan="2">
      <widget class="QLCDNumber" name="lcdNumber"/>
     </item>
     <item row="6" column="0" colspan="2">
      <widget class="QDial" name="dial"/>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>onBtn</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>clickOn()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>130</x>
     <y>149</y>
    </hint>
    <hint type="destinationlabel">
     <x>130</x>
     <y>157</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>offBtn</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>clickOff()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>244</x>
     <y>145</y>
    </hint>
    <hint type="destinationlabel">
     <x>244</x>
     <y>159</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>dial</sender>
   <signal>valueChanged(int)</signal>
   <receiver>dial</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>196</x>
     <y>400</y>
    </hint>
    <hint type="destinationlabel">
     <x>278</x>
     <y>419</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>clickOn()</slot>
  <slot>clickOff()</slot>
 </slots>
</ui>
